test/ListBinder.spec.ts

```diff
@@ -35,91 +34,106 @@ describe('ListBinder', () => {
       }
 
       class MySecondModule {
+        @ProvidesToList('myList', undefined, true)
```

Comment:
passing undefined as an argument feels a bit awkward. maybe have a default sort order if you change to enum?

Reply:
It is awkward, but it's because I don't want to include breaking changes to that point in this PR. It would mean we cannot just upgrade `saft` in our repos.

Reply:
I agree that we should change this. But just not now, since we could not easily migrate then.